pyheif/reader.py

```diff
@@ -9,17 +10,50 @@
 
 class HeifFile:
     def __init__(
-        self, *, size, data, metadata, color_profile, has_alpha, bit_depth, stride
+        self, *, size, has_alpha, bit_depth, metadata, color_profile, data, stride
     ):
         self.size = size
-        self.data = data
-        self.metadata = metadata
-        self.color_profile = color_profile
         self.has_alpha = has_alpha
         self.mode = "RGBA" if has_alpha else "RGB"
         self.bit_depth = bit_depth
+        self.metadata = metadata
+        self.color_profile = color_profile
+        self.data = data
         self.stride = stride
 
+    def __repr__(self):
+        return (
+            f"<{self.__class__.__name__} {self.size[0]}x{self.size[1]} {self.mode} "
+            f"with {str(len(self.data)) + ' bytes' if self.data else 'no'} data>"
+        )
+
+    def load(self):
+        pass  # already loaded
```

Comment:
Probably should be `return self`, to match `UndecodedHeifFile` (or return `None` there)